flair/datasets/sequence_labeling.py

```diff
@@ -435,9 +444,12 @@ def __init__(
         banned_sentences: List[str] = None,
         in_memory: bool = True,
         document_separator_token: str = None,
+        document_separator_line: str = None,
         encoding: str = "utf-8",
         skip_first_line: bool = False,
         label_name_map: Dict[str, str] = None,
+        preprocess_bioes_tags: Callable[[List[str]], List[str]] = None,
+        span_level_tag_columns: List[int] = None,
```

Comment:
This argument needs to be passed on to the three `ColumnDataset` calls below, otherwise it is not used.

Reply:
Yes, thanks for pointing that out. I missed that when transferring the code and did not spot the error: This is also the source of the issue with the parsing of the NER tags (fixed in the commits).